vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -9,12 +9,14 @@
 #include "IReactContext.h"
 
 #include <ReactUWP/Utils/ValueUtils.h>
+#include "ReactHost/MsoUtils.h"
 
 namespace winrt::Microsoft::ReactNative {
 
 ABIViewManager::ABIViewManager(
-    const std::shared_ptr<react::uwp::IReactInstance> &reactInstance,
-    const ReactNative::IViewManager &viewManager)
+    std::shared_ptr<react::uwp::IReactInstance> const &reactInstance,
+    Mso::CntPtr<Mso::React::IReactContext> const &reactContext,
+    ReactNative::IViewManager const &viewManager)
```

Comment:
woah why do we have office references in RNW code?

Reply:
Maybe because RNW initially created in Office? :) On a serious note, I had moved the IReactHost instance management from Office code to RNW. It was done as quick as possible. The code works, but I still must change namespaces and names. Any ideas/suggestions are welcome.

Reply:
As for the Mso namespace: we have published it as an open source library: https://github.com/microsoft/Mso. Again, I cut some corners and copied over the Mso code to RNW instead of using published artifacts from Mso. I am going to fix it after I implement the Mso publishing.